modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1344,6 +1489,16 @@ TEST_P(Test_ONNX_layers, DivConst)
 }
 
 
+// FIXIT disabled due to non-standard ONNX model domains, need to add ONNX domains support
+// Example:
+// DNN/ONNX: unsupported opset[1]: domain='com.microsoft.experimental' version=1
+// DNN/ONNX: unsupported opset[2]: domain='ai.onnx.preview.training' version=1
+// DNN/ONNX: unsupported opset[3]: domain='com.microsoft.nchwc' version=1
+// DNN/ONNX: unsupported opset[4]: domain='com.microsoft.mlfeaturizers' version=1
+// DNN/ONNX: unsupported opset[5]: domain='ai.onnx.ml' version=2
+// DNN/ONNX: unsupported opset[6]: domain='com.microsoft' version=1
+// DNN/ONNX: unsupported opset[7]: domain='ai.onnx.training' version=1
+#if 0
```

Comment:
Need to properly implement ONNX domains support for quantized networks.

/cc @vpisarev